src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -87,7 +87,7 @@ public void TestDriveProperties_AppContainer()
 
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Accessing drive format is not permitted inside an AppContainer.")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Accessing drive format is not permitted inside an AppContainer.")]
```

Comment:
I guess so. Still `TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot` shouldn't be present anymore as we updated `TargetFrameworkMonikers.Uap` to include UapAot flag. See: https://github.com/dotnet/buildtools/pull/1506